### PR TITLE
Update docs around the testing artifact

### PR DESCRIPTION
**Pull request by kubukoz**

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 Quick Start
 ===========
 
-circe is published to [Maven Central][maven-central] and cross-built for Scala 2.12 and 2.13,
+circe is published to [Maven Central][maven-central] and cross-built for Scala 2.12, 2.13 and 3.x,
 so you can just add the following to your build:
 
 ```scala
-val circeVersion = "0.14.1"
+val circeVersion = "0.14.10"
```
Comment (hamnis) [inline docs/quickstart.md]: perhaps we should use a banner her and refer to that instead.

`Maven Central`

then we don't have to maintain the version on release

Comment (kubukoz) [reply docs/quickstart.md]: hmm I wouldn't, you want dependency strings that are accessible I think. But we could source this info from sbt, if this is mdoc - I'll look into it

Comment (kubukoz) [reply docs/quickstart.md]: done

Comment (kubukoz) [reply docs/quickstart.md]: and I've confirmed that it does use 0.14.10 at the moment.

File: docs/optics.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ circe provides support for optics by integrating with [Monocle](https://julien-t
 dependency on `circe-optics` to your build:
 
 ```scala
-libraryDependencies += "io.circe" %% "circe-optics" % circeVersion
+libraryDependencies += "io.circe" %% "circe-optics" % "@VERSION@"
```
Comment (hamnis) [inline docs/optics.md]: circe-optics is 0.15.0
https://github.com/circe/circe-optics

Comment (hamnis) [inline docs/optics.md]:
```suggestion
libraryDependencies += "io.circe" %% "circe-optics" % "0.15.0"
```

Comment (kubukoz) [reply docs/optics.md]: ohh I forgot it's a separate repo. I'll hardcode then, I guess

Comment (kubukoz) [reply docs/optics.md]: used the version we use in build.sbt to add the dependency for docs. It inlines as 0.15.0:

image

Comment (kubukoz) [reply docs/optics.md]: used the value from `docs/libraryDependencies`

**Review by hamnis — APPROVED** [review]
thanks

**Pull request merged**